packages/twenty-server/src/engine/core-modules/auth/auth.resolver.ts

```diff
@@ -490,6 +490,8 @@ export class AuthResolver {
     @AuthUser() currentUser: User,
     @AuthProvider() authProvider: AuthProviderEnum,
   ): Promise<SignUpOutput> {
+    await this.signInUpService.isWorkspaceCreationAllowed(currentUser);
```

Comment:
Naming is a bit strange because we expect it to return a boolean but it's more like checkWorkspaceCreationIsAllowedOrThrow